libraries/chef_package_helper.rb

```diff
@@ -43,6 +43,7 @@ def install_product(resource)
     #
     #
     def reconfigure_product(resource)
+      install_mixlib_versioning
       chef_ingredient resource.product_name do
         action :reconfigure
```

Comment:
My thinking was that recipes would typically install a package using chef_ingredient before attempting to reconfigure.  However, I am using `package` to install a `remote_file` and I don't want to deal with the `product_lookup` logic.  Later I reconfigure that product and I was having to call `install_mixlib_versioning()`in my wrapper cookbook.  I would like to see this stay.